container.go

```diff
@@ -32,12 +32,6 @@ type ContainerConfig struct {
 	// RootFs is the container workload image on the host.
 	RootFs string
 
-	// Interactive specifies if the container runs in the foreground.
-	Interactive bool
-
-	// Console is a console path provided by the caller.
-	Console string
-
```

Comment:
You're right, we can do per container tty console, but I noticed some errors in the code, preventing us from doing such a thing. Everything was pod oriented, meaning in case of several containers, it would not have worked as expected. And in the same time, I have seen that interactive was specified at container level but it should be specified at pod level. I was thinking about fixing the tty console for each container in a future commit.
But I can also remove what I have removed and instead, I can fix it in another commit later (I mean the console thing), but we keep the Interactive change.